ppr-ui/tests/unit/SecuredPartyValidation.spec.ts

```diff
@@ -105,11 +105,11 @@ describe('Secured Party validation tests - individual', () => {
     expect(radios.length).toBe(2)
     radios.at(0).trigger('click')
     await Vue.nextTick()
-    wrapper.find('#txt-first').setValue('first')
-    wrapper.find('#txt-last').setValue('person')
+    wrapper.find('#txt-first-party').setValue('first')
+    wrapper.find('#txt-last-party').setValue('person')
     
-    wrapper.find('#txt-email').setValue('person@')
-    wrapper.find('#txt-email').trigger('blur')
+    wrapper.find('#txt-email-party').setValue('person@')
+    wrapper.find('#txt-email-party').trigger('blur')
```

Comment:
would be nice if all these ids were declared as selectors up top (same as other PR), but fine for now